modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/CommentResourceImpl.java

```diff
@@ -751,45 +726,16 @@ private DLFileEntry _getDLFileEntry(
 			String externalReferenceCode, Long siteId)
 		throws Exception {
 
-		DLFileEntry dlFileEntry =
-			_dlFileEntryService.fetchFileEntryByExternalReferenceCode(
-				siteId, externalReferenceCode);
-
-		if (dlFileEntry == null) {
-			StringBundler sb = new StringBundler(4);
-
-			sb.append("No document exists with external reference code ");
-			sb.append(externalReferenceCode);
-			sb.append(" and site ID ");
-			sb.append(siteId);
-
-			throw new NotFoundException(sb.toString());
-		}
-
-		return dlFileEntry;
+		return _dlFileEntryService.getFileEntryByExternalReferenceCode(
+			siteId, externalReferenceCode);
 	}
 
 	private JournalArticle _getLatestJournalArticle(
```

Comment:
Does it make sense to have this method that just makes a request to another service and do nothing else?